admin_manual/installation/source_installation.rst

```diff
@@ -398,7 +398,7 @@ your ``php-fpm`` configuration and increase the ``upload_max_filesize`` and
 ``post_max_size`` values. You will need to restart ``php5-fpm`` and your HTTP 
 server in order for these changes to be applied.
 
-**.htaccess notes for Web servers \<> Apache**
+**.htaccess notes for Apache**
```

Comment:
This heading used to say it was related to Web servers other than Apache. Now it says "for Apache".
Is that correct?
@carlaschroder